extensions/amp-script/amp-script.md

```diff
@@ -415,6 +415,9 @@ The value of `max-age` should be chosen carefully:
 
 If you don't publish signed exchanges, `max-age` does nothing.
 
+### nodom
```

Comment:
thanks!

Reply:
Still wondering how it didn't propagate into my branch...

Reply:
and here i am just impressed that you realized it before merging :)